jinahub/indexers/query/vector/FaissIndexer/__init__.py

```diff
@@ -114,36 +174,60 @@ def build_advanced_index(self, vecs: 'np.ndarray'):
         index.nprobe = self.nprobe
         return index
 
-    @batching
     def _build_partial_index(self, vecs: 'np.ndarray', index):
         vecs = vecs.astype(np.float32)
         if self.normalize:
             from faiss import normalize_L2
+
             normalize_L2(vecs)
         index.add(vecs)
 
-    def query(self, vecs: 'np.ndarray', top_k: int, *args, **kwargs) -> Tuple['np.ndarray', 'np.ndarray']:
+        return
+
+    @requests(on='/search')
+    def query(self, docs: DocumentArray, parameters: Dict = None, *args, **kwargs):
```

Comment:
I am not sure if it matches the `signature` but maybe add `Optional` to parameters type hint?

Reply:
Done